ivadomed/utils.py

```diff
@@ -35,6 +35,12 @@ def __str__(self):
 
 
 def initialize_wandb(wandb_params):
+
+    # If key is not defined or is empty, exit immediately.
+    # https://github.com/ivadomed/ivadomed/issues/1252
+    if wandb_params[WandbKW.WANDB_API_KEY] == '':
+        return False
```

Comment:
Perhaps nitpicky, but this:
1. doesn't inform the user about wandb being disabled and the reason. 
2. Evaluating an empty string as `== ''` doesn't help in avoiding instances with whitespace (ref: https://stackoverflow.com/a/9573278) 
```
 >>> bool("")
 False
 >>> bool("   ")
 True
 >>> bool("   ".strip())
 False
```
Therefore, I'm inclined to something along the lines of which takes care of that:

```
try:
  if wandb_params[WandbKW.WANDB_API_KEY].strip():
    raise ValueError()

  wandb.login(...)
```

Reply:
sounds good! can you please do all the changes you proposed in your review? thanks!